ProjectWithUMLDiagram/src/com/beyond/person/core/model/BasePerson.java

```diff
@@ -0,0 +1,95 @@
+package com.beyond.person.core.model;
+
+import com.beyond.person.core.enums.Gender;
+import com.beyond.person.core.exceptions.*;
+import com.beyond.person.core.helper.Help;
+
+public class BasePerson {
+    protected String name;
+    protected String lastName;
+    protected String designation;
+    protected String age;
+    protected String gender;
+    protected String email;
+    protected String nickName;
+
+
+    /**
+     * constructor for BasePerson
+     *
+     */
+    public BasePerson(String name, String lastName){
+            if (name.length() > 0 && name.length() < 255 && 0 < lastName.length() && lastName.length() < 255 && (Help.checkAlphabet(name)) && (Help.checkAlphabet(lastName))) {
```

Comment:
@AraChilingaryan write validator for this

Reply:
done